src/common/urls.py

```diff
@@ -10,13 +10,13 @@
         name="custom_field_form",
     ),
     path(
-        "custom_field-create/",
+        "custom-field-create/",
```

Comment:
@loststoner  Let's change this into "custom-fields-create" to be consistent with the other path declarations.